From 14cb9c8800e3a74c8af849788c79765279cf3286 Mon Sep 17 00:00:00 2001
From: Bogdan Hamciuc <bogdan.hamciuc@nxp.com>
Date: Mon, 7 Jan 2019 15:51:45 +0200
Subject: [PATCH 009/269] s32g: Update includes after upstream master pull

A couple of included headers are now under include/common.

Upstream-Status: Pending 

Signed-off-by: Bogdan Hamciuc <bogdan.hamciuc@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
Issue: ALB-3452
---
 plat/s32g/s32g275_bl31.c | 2 +-
 plat/s32g/s32g_psci.c    | 2 +-
 plat/s32g/s32g_xrdc.c    | 2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/plat/s32g/s32g275_bl31.c b/plat/s32g/s32g275_bl31.c
index 1b5d5e44f..33bd1cb0c 100644
--- a/plat/s32g/s32g275_bl31.c
+++ b/plat/s32g/s32g275_bl31.c
@@ -6,7 +6,7 @@
 
 #include <arch_helpers.h>
 #include <assert.h>
-#include <bl_common.h>
+#include <common/bl_common.h>
 #include <psci.h>
 #include <drivers/arm/gicv3.h>
 #include "platform_def.h"
diff --git a/plat/s32g/s32g_psci.c b/plat/s32g/s32g_psci.c
index 1702c9116..99850b70b 100644
--- a/plat/s32g/s32g_psci.c
+++ b/plat/s32g/s32g_psci.c
@@ -5,7 +5,7 @@
  */
 #include <string.h>
 #include <assert.h>
-#include <debug.h>	/* printing macros such as INFO() */
+#include <common/debug.h>	/* printing macros such as INFO() */
 #include "platform_def.h"
 
 IMPORT_SYM(unsigned long, __BL31_START__, bl31_start);
diff --git a/plat/s32g/s32g_xrdc.c b/plat/s32g/s32g_xrdc.c
index da27af8a8..b0afd8c03 100644
--- a/plat/s32g/s32g_xrdc.c
+++ b/plat/s32g/s32g_xrdc.c
@@ -8,7 +8,7 @@
  * in a secure DDR area.
  */
 #include <lib/mmio.h>
-#include <debug.h>
+#include <common/debug.h>
 #include "platform_def.h"
 
 /* Master Domains - application core, safety core, jtag, GIC, GMAC, eDMA ... */
-- 
2.17.1

